protos/service.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package tasktimer;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/nivista/tasktimer/.gen;pb";
+
+service TaskTimer {
+    rpc CreateTimer (CreateTimerRequest) returns (CreateTimerResponse) {}
+
+    rpc DeleteTimer (DeleteTimerRequest) returns (DeleteTimerResponse) {}
+
+    rpc GetTimer (GetTimerRequest) returns (GetTimerResponse) {}
+}
+
+message CreateTimerRequest {
+    string account = 1;
```

Comment:
I prefer the name namespace, domain or tenant here.

Reply:
awesome, i'll go with domain